src/view/configuration/configuration.jsx

```diff
@@ -230,457 +232,456 @@ const validationSchema = object()
-                              >
-                                Any rule components or data elements using this
-                                instance will no longer function as expected
-                                when running on your website. We recommend
-                                removing or updating those resources before
-                                publishing your next library.
-                              </Alert>
-                            ) : null}
-                            <div />
-                          </div>
-                          <div className="u-gapTop">
-                            <InfoTipLayout tip="Your assigned config ID, which links the SDK to the appropriate accounts and configuration.">
-                              <FieldLabel
-                                labelFor="configIdField"
-                                label="Config ID"
-                              />
-                            </InfoTipLayout>
-                            <div>
-                              <WrappedField
-                                id="configIdField"
-                                name={`instances.${index}.configId`}
-                                component={Textfield}
-                                componentClassName="u-fieldLong"
-                                supportDataElement="replace"
-                              />
-                            </div>
-                          </div>
-                          <div className="u-gapTop">
-                            <InfoTipLayout tip="Your assigned Experience Cloud organization ID.">
-                              <FieldLabel
-                                labelFor="orgIdField"
-                                label="IMS Organization ID"
-                              />
-                            </InfoTipLayout>
-                            <div>
-                              <WrappedField
-                                id="orgIdField"
-                                name={`instances.${index}.orgId`}
-                                component={Textfield}
-                                componentClassName="u-fieldLong"
-                                supportDataElement="replace"
-                              />
-                              <Button
-                                id="orgIdRestoreButton"
-                                label="Restore to default"
-                                onClick={() => {
-                                  const instanceDefaults = getInstanceDefaults(
-                                    initInfo
-                                  );
-                                  setFieldValue(
-                                    `instances.${index}.orgId`,
-                                    instanceDefaults.orgId
-                                  );
-                                }}
-                                quiet
-                                variant="quiet"
```

Comment:
This is what the react-spectrum warning was about. Removing the `variant` attribute fixed the issue. The `quiet` attribute we already had provides appropriate "quiet" styling.